### PR TITLE
Added "Starting a server (Advanced)"

### PR DESCRIPTION
**Pull request by Sweet-Tan**
Included arguments, example commands, and default config.ini location for Windows.

### CODE REVIEW

File: playing/multiplayer/index.rst [hunk 1]
```diff
@@ -13,9 +13,32 @@ When you open the multiplayer window you will see a list of public servers and s
...
+OpenRCT supports headless mode, useful for dedicated servers. You can enable headless mode by running: 
+
+``openrct2 host <saved game path> --headless``
+
+Additional arguments can be used to change the behavior of the server.
+
+=================   ===========================  ==================================================
+    Argument                 Variable                  Description
+=================   ===========================  ==================================================
+--console           N/A                          Keeps the console window open
```
Comment (janisozaur) [inline playing/multiplayer/index.rst]: This option is only present on Windows and not needed on other platforms, where `stdout` and `stderr` is always available.

File: playing/multiplayer/index.rst [hunk 2]
```diff
@@ -13,9 +13,32 @@ When you open the multiplayer window you will see a list of public servers and s
 If you want to join a server that is not advertised on the public server list, you must click the "New server" button and enter the hostname or IP address (and optionally a port if the server is not using the default OpenRCT2 port, 11753). Upon doing this, the server will be saved to your server list so you do not have to type the address in every time you want to join it.
 
 
-Starting a server
+Starting a server
```
Comment (janisozaur) [inline playing/multiplayer/index.rst]: Trailing whitespace

File: playing/multiplayer/index.rst [hunk 3]
```diff
@@ -13,9 +13,32 @@ When you open the multiplayer window you will see a list of public servers and s
...
+Additional arguments can be used to change the behavior of the server.
+
+=================   ===========================  ==================================================
+    Argument                 Variable                  Description
+=================   ===========================  ==================================================
+--console           N/A                          Keeps the console window open
+--headless          N/A                          Runs OpenRCT without graphical interface
+--port              Port Number                  Changes the server port number
+--user-data-path    Path to config.ini folder    Changes the server configuration data path
+--verbose           N/A                          Output additional information to the console
```
Comment (janisozaur) [inline playing/multiplayer/index.rst]: There are more options than the list provided, add also at least `--help`. Also put options in backticks.

**Comment by Gymnasiast** [comment]
This _has_ to be the longest time a PR has been open and was still merged. ;)

**Pull request merged**